File: Data Structures/Linked Lists/Insert a node at the head of a linked list .cpp
/*
   Contributer : github.com/Blind Bluffer
   Email : shivank309@gmail.com
*/
/*
  Insert Node at the begining of a linked list
  Initially head pointer argument could be NULL for empty list
  Node is defined as 
  struct Node
  {
     int data;
     struct Node *next;
  }
return back the pointer to the head of the linked list in the below method.
*/
Node* Insert(Node *head,int data)
{
  // Complete this method
  Node* n=new Node;
  n->data=data;
  if(head==NULL){
  n->next=NULL;
  head=n;
  return head;
  }
  else{
  n->next=head;
  head=n;
  }
return head;
}

File: Data Structures/Linked Lists/Insert a node at the head of a linked list.java
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
*/
Node Insert(Node head,int x) {
    

  
  Node n=new Node();
  n.data=x;
  if(head==null){
  n.next=null;
  head=n;
  return head;
  }
  else{
  n.next=head;
  head=n;
  }
return head;
}
